test/matrix/csr_kernels2.cpp

```diff
@@ -34,7 +34,7 @@ class Csr : public CommonTestFixture {
 protected:
     using Arr = gko::array<int>;
     using Vec = gko::matrix::Dense<value_type>;
-    using Vec2 = gko::matrix::Dense<gko::next_precision<value_type>>;
+    using Vec2 = gko::matrix::Dense<gko::next_precision_base<value_type>>;
```

Comment:
```suggestion
    using Vec2 = gko::matrix::Dense<gko::next_precision<value_type>>;
```
Although I think this is not used at all. (But that is nothing to change in this PR.)

Reply:
It is the same reason for keeping that. the value_type will be float with enabling dpcpp_single_mode.
I would rather than deleting them because they are not used.